utils/generateImplementedTestCases.pl

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env perl
+
+use 5.16.0;
+use warnings;
+use Zonemaster::Engine;
+use File::Basename;
+
+# page header
+print '<!-- File generated by ', basename ($0), ", script in zonemaster/zonemaster util directory.\n";
+print "Use that script to generate a new file for each release of Zonemaster when \n";
+print "Zonemaster-Engine also has been updated.-->\n\n";
+print "# Implemented Test Cases\n\n";
```

Comment:
IMO it would be better to just call this Test Cases. The word "implemented" adds more confusion than clarity. We shouldn't be documenting unimplemented test cases in the master branch anyway. By extension I think the word implemented should be removed every from this PR.

Reply:
Test cases appear in two shapes in Zonemaster, as test case specifications in Zonemaster/Zonemaster and as implemented test cases (test cases in code) in Zonemaster-Engine. It is that distinction show. In Zonemaster/Zonemaster, as the generated document will appear in, test case usually refer to the specification.

If you could suggest something else that clarifies that this does not refer to the specification I could use that instead.

Reply:
My suggestion is that we never release changes to the specifications without the corresponding functionality in the implementation.

Having a set of defined test case specifications and a page of links to implemented subset of test cases would work for the situation where we add new test cases - though I don't see how this model benefits our users. But this model breaks down when it comes to changes to existing test cases. A more reasonable model is, IMHO, to merge spec and implementation at the same time.

Reply:
> My suggestion is that we never release changes to the specifications without the corresponding functionality in the implementation.

I agree with you, but don't you miss my point that I want to make a distinction between test case specification and test case implementation in Perl code. Or do I misunderstand you?